v22.2/performance-recipes.md

```diff
@@ -64,6 +64,11 @@ This section describes how to use CockroachDB commands and dashboards to identif
     <td><ul><li>Your tables have long write times.</li></ul></td>
     <td><ul><li><a href="#slow-writes">Remove unnecessary indexes.</a></li></ul></td>
   </tr>
+  <tr>
+    <td><ul><li>You experience high latency on queries that cannot be explained by high contention or a suboptimal query plan. You might also see high CPU on one or more nodes.</li></ul></td>
+    <td><ul><li>You are scanning over large numbers of <a href="architecture/storage-layer.html#mvcc">MVCC versions</a>. This is similar to how a full table scan can be slow.</li></ul></td>
```

Comment:
I believe high latency + high CPU can be symptoms of other factors as well, such as workload concurrency exceeding CPUs. This is documented at https://www.cockroachlabs.com/docs/v22.1/common-issues-to-monitor.html#workload-concurrency - is it reasonable to cross-link to that section, and/or to mitigate the diagnosis with "You may be scanning over large numbers..."?